src/yukon/backend/requirements.txt

```diff
@@ -1,5 +1,28 @@
-# Development-only dependencies (not required to use the library)
+aiofiles==0.5.0
```

Comment:
I am not certain about this change. Check: https://github.com/UAVCAN/Yukon/pull/26#discussion_r405675344

Reply:
As I understand, right now tox is used as tool-for-everything: run tests, make packages, use 2 versions of python and so on. It's not about bootstrapping dev environment, it's about CI. Also, it does not manage deps right now - pyuavcan and nunavat should be installed manually.

Do we really need to depend on current configuration of tox, that is not configured to be used as dependency manager right now. Or should we use standard python way? I think we need to ask person who knows Python better. And then decide what to do: make tox configuration correct or use pip/pipenv or somthing else.

Reply:
I agree about the bootstraping part. Accepting now but I am going to request @thirtytwobits opinion here.